packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonDot.story.tsx

```diff
@@ -0,0 +1,62 @@
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+
+import { RadioButtonDot } from './RadioButtonDot';
+
+export default {
+  title: 'Forms/RadioButtonList/Dots',
+  component: RadioButtonDot,
+} as Meta;
+
+const Wrapper: React.FC<{ title: string }> = ({ title, children }) => (
```

Comment:
Idea: Why don't we create a proper `<RadioButtonList.Group>` component (that's just a `div aria-role="radiogroup"` and export that component so we can use it in the UI too.

Reply:
Hmm, maybe I should just add `aria-role="radiogroup"` to the `RadioButtonList` component? Is it what you mean?

Reply:
Oh right, we can just add it there – great suggestion 👍